metadata-db-app/src/migrations/062_add_variable_constraint.clj

```diff
@@ -0,0 +1,54 @@
+(ns migrations.062-add-variable-constraint
+  (:require
+   [config.mdb-migrate-helper :as h]))
+
+(def ^:private table-name "cmr_variables")
+(def ^:private table (format "METADATA_DB.%s" table-name))
+(def ^:private new-constraint-name "variables_con_n_r_p")
+(def ^:private new-index-name "variables_idx_n_r_p")
+(def ^:private old-constraint-name "variables_con_rev")
+(def ^:private old-index-name "variables_ucr_i")
+
+(defn- add-variable-constraint
+  []
+  (h/sql
+   (format (str "ALTER TABLE %s "
```

Comment:
It would be so much easier to just read the sql statement as it is, rather than going through the variable lookup. Just my 2 cents. You can keep it the way it is if you feel strongly about it.